roles/neutron-data-network/tasks/monitoring.yml

```diff
@@ -28,3 +28,14 @@
 - name: l3 router check
   sensu_check: name=check-l3-router plugin=check-l3-router.sh use_sudo=true
   notify: restart sensu-client
+
+- name: default route gateway
+  shell: ip route | awk "/default.+{{ hostvars[inventory_hostname][primary_interface].device }}/ { print \$3; exit }"
+  register: default_route_gateway
+  ignore_errors: True
+
+- name: gateway ip duplicate check
+  sensu_check: name=check-gateway-ip-duplicate plugin=check-arping.sh
+               args="-I {{ hostvars[inventory_hostname][primary_interface].device }} -d {{ default_route_gateway.stdout }}"
+  notify: restart sensu-client
+  when: default_route_gateway.stdout|match('\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}')
```

Comment:
I was checking it's a valid IP just in case there was a problem running the previous command that gets the gateway IP. Took it out as we now use ansible generated facts as you suggested.